channeldb/graph_test.go

```diff
@@ -1942,10 +1942,23 @@ func TestFilterKnownChanIDs(t *testing.T) {
 
 	// If we try to filter out a set of channel ID's before we even know of
 	// any channels, then we should get the entire set back.
+	currentTimeStamp := time.Now()
```

Comment:
style nit: should squash this commit into the commit that changes the behaviour (ie, where the unit tests starts failing)

Reply:
ok squashed the logic change into one commit because changes were all in the same file.